test/Common/Helpers/AssertHelper.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Linq;
+using System.Linq.Expressions;
```

Comment:
My favorite namespace 😄

Reply:
I thought you'd like that. I'm actually pretty happy with the expression, too. I'm using it to make a matcher for NSubstitute that turned out to be pretty useful for this set of tests

Reply:
Yeah they look really good.